packages/common/commonTypes/src/index.ts

```diff
@@ -12,3 +12,11 @@ export interface PSUDataItem {
   ApplicationName: string
   ExpiryTime: number
 }
+
+export interface NotifyDataItem {
```

Comment:
This is a minimal version of the `PSUDataItem` data type, so we don't pass around sensitive data without a reason.